docs/pages/get_started/developers/releases.js

```diff
@@ -38,6 +38,24 @@ components,
 `}
       />
 
+      <MainSection
+        name="Alpha releases"
+        description={`Gestalt's deployment system supports automatic alpha releases.
+
+To run an alpha release, follow these steps:
+
+- Create a branch (regular step)
+- Submit a Pull Request to github (regular step)
+- Click on the "Compare & pull request" on Github
+- Instead of <code>base repository:pinterest/gestalt</code> and <code>base: master</code>, select <code>base: alpha</code>
+- Merge your branch. There's no need to set a label or wait for builds to pass as some might fail.
```

Comment:
Oh yikes. This is pretty worrisome. @rlingineni can you keep working on this pipeline to make sure that builds pass CI before they're published?